lnrpc/neutrinorpc/neutrino.proto

```diff
@@ -0,0 +1,214 @@
+syntax = "proto3";
+
+package neutrinorpc;
+
+option go_package = "github.com/lightningnetwork/lnd/lnrpc/neutrinorpc";
+
+// Neutrino is a service that can be used to get information about the current
+// state and scan for transactions.
+service NeutrinoKit {
+    /*
+    Status returns the status of the light client neutrino instance,
+    along with height and hash of the best block, and a list of connected
+    peers.
+    */
+    rpc Status (StatusRequest) returns (StatusResponse);
+
+    /*
+    AddPeer adds a new peer that has already been connected to the server.
+    */
+    rpc AddPeer (AddPeerRequest) returns (AddPeerResponse);
```

Comment:
ListPeers?

Reply:
I can add that if you want. In this pr, the status call lists peers right now.